build/mixxx.py

```diff
@@ -323,11 +323,13 @@ def setup_osx_platform_sdk(self):
         sysroot = Script.ARGUMENTS.get('sysroot', '')
         if sysroot:
             self.env.Append(CCFLAGS=['-isysroot', sysroot])
+            self.env.Append(LINKFLAGS=['-isysroot', sysroot, '-Wl,-syslibroot,' + sysroot])
+            self.env.Append(OSX_SYSROOT=sysroot)
 
         # If no sysroot was specified, pick one automatically. The only platform
         # we pick one automatically on is OS X.
         if self.platform_is_osx:
-            if '-isysroot' in self.env['CXXFLAGS'] or '-isysroot' in self.env['CFLAGS']:
+            if '-isysroot' in self.env['CXXFLAGS'] or '-isysroot' in self.env['CFLAGS'] or '-isysroot' in self.env['CCFLAGS']:
```

Comment:
Did you find this was necessary? I believe CXXFLAGS should have already covered CCFLAGS.

```
CXXFLAGS

    General options that are passed to the C++ compiler. By default, this includes the value of $CCFLAGS, so that setting $CCFLAGS affects both C and C++ compilation. If you want to add C++-specific flags, you must set or override the value of $CXXFLAGS. 
```
https://scons.org/doc/1.2.0/HTML/scons-user/a4774.html